xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -33,9 +33,8 @@ using namespace jni;
 
 using namespace std::chrono_literals;
 
-// those are empirical values while the HD buffer
-// is the max TrueHD package
-const unsigned int MAX_RAW_AUDIO_BUFFER_HD = 61440;
+// those are empirical values
+const unsigned int MAX_RAW_AUDIO_BUFFER_HD = 32768;
```

Comment:
constexpr and would be nice if all these "defines" could be in an anonymous namespace. 😊 Maybe for a future cleanup.